packages/perseus/src/widgets/group.jsx

```diff
@@ -69,7 +123,9 @@ class Group extends React.Component<Props> {
         // give ourselves number -1. To combat this, we forceUpdate in
         // componentDidMount so that we can number ourselves properly. But,
         // really we should have a more unidirectional flow. TODO(marcia): fix.
-        const number = _.indexOf(this.props.findWidgets("group"), this);
+        const groupWidgets: $ReadOnlyArray<Group> =
```

Comment:
Pedantically, the return value here would be `$ReadOnlyArray<Widget>`. Although the filter is using a widget type, we'd probably be safer type this as Widget to be explicit and avoid someone changing the `findWidgets` call in a way that returns a mixture. Maybe I'm being paranoid.

Reply:
Fixed